### ISSUE TITLE
[fr] feedback from a user "et" vs "est"

### ISSUE THREAD

**Issue by tiff** [languagetool-org/languagetool#3822]
> "Ceci et un essai": c’est faux, ce devrait être "ceci est un essai".

**Comment by udomai** [comment]
As an experiment, I have written a rule that looks for "et" in sentences that do not contain any finite verb. This results in a great many false positives in sentences that are okay without verb: _Et voilà._
Especially in headlines, you will find a lot of sentences without finite verb, containing "et".

I will try if a sequence of antipatterns gets this broad approach anywhere. Otherwise, we might have to restrict this to more specific contexts.

@vkyfox, any other ideas how one could go about his?

**Comment by dpelle** [comment]
Either can be grammatically correct.  They mean something different.

Admittedly, it's less likely that someone would write "ceci et un essai"
but there is way to check what the user intended to say.

I would prefer to not report errors in such cases to avoid false positives.

That said, there can be cases where we can be more certain that it'
a mistake such as   "Elle et grande.".   "On et grands", "Il et grand".
I just checked and LT already signals an error for the last 2 sentences.
It does not  signal an error for "Elle et grande.", presumably because
"et" can be after "elle" in sentences like "Avec elle et quelqu'un d'autre".

**Comment by udomai** [comment]
@dpelle, that's quite right. I also think that the user's sentence is constructed, since it literally says "this is a test". They possibly did not think of contexts like the below:

_Entre **ceci et une célèbre publicité** un peu plus récente, cet animal a fini par conquérir les cœurs de tous les amateurs de NAC._

However, the confusion of "et" and "est" (which are homophonous in many parts of France) is definitely something we want to spot, and not only in three-word sentences. A French user who types "Kepler **et** mon héros favori et mes romans parlent effectivement de la vie de Kepler." will definitely be thankful for us suggesting "est", even though the sentence is grammatically correct with "et".

I want to know what XML rules can do in this regard. Your suggestion that "on" and "il" are quite improbable before "et" is an excellent suggestion!
